### PR TITLE
Manual Triggers

### PR DESCRIPTION
**Pull request by assignUser**
This PR adds manual triggers for the packaging and report workflows. They can now be started with a custom ISO date string (e.g. to re-run failed jobs from a specific day) , this will only be active once this is merged to master.
I also removed the `on.push` triggers as they are superfluous with manual triggers imho and could lead to unwanted triggering of the workflows.

### CODE REVIEW

File: .github/workflows/nightly_packaging_report.yml [hunk 1]
```diff
@@ -1,9 +1,13 @@
 name: Report packaging
 
 on:
-  push:
-    paths:
-      - '.github/workflows/nightly_packaging_report.yml'
+  workflow_dispatch:
+    inputs:
+      date:
+        description: 'ISO date string to use in the action.'
```
Comment (raulcd) [inline .github/workflows/nightly_packaging_report.yml:7]: Can we add an example? I am thinking on something like:
`ISO date string to use in the action, i.e: 2020-01-24` or `ISO date string to use in the action. Format YYYY-MM-DD`
I understand that ISO date is already this format but I think it's a nice reminder.

**Review by raulcd — COMMENTED** [review]
LGTM just a minor comment on the description fields for the date.

File: .github/workflows/nightly_packaging_report.yml (hunk 1 above)
Comment (assignUser) [reply .github/workflows/nightly_packaging_report.yml:7]: Yep, good idea!

Comment (assignUser) [reply .github/workflows/nightly_packaging_report.yml:7]:
```suggestion
        description: 'ISO date string to use in the action. Format YYYY-MM-DD'
```

File: .github/workflows/nightly_packaging_submit.yml [hunk 2]
```diff
@@ -1,9 +1,13 @@
 name: Submit packaging
 
 on:
-  push:
-    paths:
-      - '.github/workflows/nightly_packaging_submit.yml'
+  workflow_dispatch:
+    inputs:
+      date:
+        description: 'ISO date string to use in the action.'
```
Comment (raulcd) [inline .github/workflows/nightly_packaging_submit.yml:7]:
```suggestion
        description: 'ISO date string to use in the action. Format YYYY-MM-DD'
```

File: .github/workflows/nightly_release_report.yml [hunk 3]
```diff
@@ -1,9 +1,13 @@
 name: Report release
 
 on:
-  push:
-    paths:
-      - '.github/workflows/nightly_release_report.yml'
+  workflow_dispatch:
+    inputs:
+      date:
+        description: 'ISO date string to use in the action.'
```
Comment (raulcd) [inline .github/workflows/nightly_release_report.yml:7]:
```suggestion
        description: 'ISO date string to use in the action. Format YYYY-MM-DD'
```

File: .github/workflows/nightly_tests_submit.yml [hunk 4]
```diff
@@ -1,9 +1,13 @@
 name: Submit tests
 
 on:
-  push:
-    paths:
-      - '.github/workflows/nightly_tests_submit.yml'
+  workflow_dispatch:
+    inputs:
+      date:
+        description: 'ISO date string to use in the action.'
```
Comment (raulcd) [inline .github/workflows/nightly_tests_submit.yml:7]:
```suggestion
        description: 'ISO date string to use in the action. Format YYYY-MM-DD'
```

**Review by raulcd — APPROVED** [review]

**Pull request merged**